scripts/install-linux.sh

```diff
@@ -13,6 +13,15 @@ else
 fi
 echo "Detected OS is $OS_TYPE"
 
+
+# Check CPU architecture
+CPU_ARCH=$(uname -p)
```

Comment:
As noted in previous comment 
uname -p resolves to x86_64 to arm64

Reply:
Fixed with aaa78f8